<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementEditor</class>
 <widget class="QWidget" name="ElementEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_labels">
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditElementLabel"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_inputs">
     <property name="text">
      <string>Input Ports:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_frequency">
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_color">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxFrequency">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="comboBoxColor">
     <item>
      <property name="text">
       <string>Blue</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/output/resources/output/BlueLedOff.svg</normaloff>:/output/resources/output/BlueLedOff.svg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/output/resources/output/GreenLedOff.svg</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Purple</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/output/resources/output/PurpleLedOff.svg</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/output/resources/output/RedLedOff.svg</normalon>
       </iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>White</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/output/resources/output/WhiteLedOff.svg</normalon>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" rowspan="8">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBoxInputSz"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
